.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+.env
```

Comment:
You are supposed to provide a `.env-sample` file so that other devs cloning this repo know how to create the `.env` file needed for it to run. This project won't run on my machine since I don't have a `.env` and with no `.env-sample` file it means I have to dig through the code to find all ENV variables that I need to add to it.